src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,87 @@
-import React from 'react';
+import React, { FC, useEffect, useState } from 'react';
+import classNames from 'classnames';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieData } from '../../types/MovieData';
+import { ResponseError } from '../../types/ReponseError';
+import { MovieCard } from '../MovieCard';
+
+interface Props {
+  onAdd: (movie: Movie) => void;
+}
+
+export const FindMovie: FC<Props> = ({ onAdd }) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [searchedQuery, setQuery] = useState('');
+  const [isError, setIsError] = useState(false);
+  const [isSubmitted, setSubmitted] = useState(false);
+
+  const isValidData = (data: MovieData | ResponseError): data is MovieData => {
+    return (data as MovieData).Title !== undefined;
+  };
+
+  useEffect(() => {
+    if (searchedQuery) {
+      setIsError(false);
+    }
+  }, [searchedQuery]);
+
+  const findMovie = async () => {
+    if (!searchedQuery) {
```

Comment:
don't send a request if you have an invalid `title`, just with spaces, for example
Also - put an error like `Put the movie name`